tools/webpack-core/src/utils/resolve-recurse/resolve.js

```diff
@@ -0,0 +1,144 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+//resolve.js
+
+var Promise = require('promise');
+var findup = Promise.denodeify(require('findup'));
+var resolve = Promise.denodeify(require('resolve'));
+var isCore = require('is-core-module');

```

Comment:
You should also be able to just cross-reference stuff with [module.builtinModules](https://nodejs.org/docs/latest-v12.x/api/module.html#module_module_builtinmodules) instead of using `is-core-module`.